tools/ManifestSignerClient/CertificateGenerator/Gen_RootCA_with_Signer_ECDsa.bat

```diff
@@ -0,0 +1,27 @@
+REM ECDSA scripts
```

Comment:
Are we assuming customers to run the script on windows only?

Reply:
Given the script depends on openssl, customers need to install it that we need to share the info. in the readme.